web/src/labs/marked/index.ts

```diff
@@ -26,7 +26,7 @@ export const marked = (markdownStr: string, blockParsers = blockElementParserLis
   let matchedInlineParser = undefined;
   let matchedIndex = -1;
 
-  for (const parser of inlineElementParserList) {
+  for (const parser of inlineParsers) {
```

Comment:
@boojack Is this a mistake? If we use `inlineElementParserList` here, `inlineParsers` (which are passed in many files, like [Bold parser](https://github.com/usememos/memos/blob/main/web/src/labs/marked/parser/Bold.ts#L13)) will never work.

Reply:
Thanks, good catch!